kythe/release/appengine/buildbot/master/master.cfg.template

```diff
@@ -42,29 +42,37 @@ c['protocols'] = {'pb': {'port': 9989}}
 
 ####### CHANGESOURCES
 
-# List of Pull Request authors that automatically trigger builds
-autoBuildUsers = [
-    'creachadair',
-    'danielmoy-google',
-    'jrtom',
-    'salguarnieri',
-    'schroederc',
-    'shahms',
-    'zrlk',
-]
-def pullRequestFilter(req):
+def pullRequestFilter(pr):
```

Comment:
I guess it's probably consistent with the existing code, but `snake_case` is [more conventional][pep8] for Python. (Probably it makes sense to be consistent for now, but I wanted to point it out). 

[pep8]: https://www.python.org/dev/peps/pep-0008/?#function-and-variable-names

Reply:
Ack.